README.md

```diff
@@ -1,9 +1,11 @@
 # Elucidation
 
-[![pipeline status](https://gitlab.com/fortitudetec/elucidation-project/elucidation/badges/master/pipeline.svg)](https://gitlab.com/fortitudetec/elucidation-project/elucidation/-/commits/master)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.fortitudetec%3Aelucidation&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.fortitudetec%3Aelucidation)
-[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.fortitudetec%3Aelucidation&metric=coverage)](https://sonarcloud.io/dashboard?id=com.fortitudetec%3Aelucidation)
+[![Build](https://github.com/elucidation-project/elucidation/workflows/build/badge.svg)](https://github.com/elucidation-project/elucidation/actions?query=workflow%3Abuild)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=elucidation-project_elucidation&metric=alert_status)](https://sonarcloud.io/dashboard?id=elucidation-project_elucidation)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=elucidation-project_elucidation&metric=coverage)](https://sonarcloud.io/dashboard?id=elucidation-project_elucidation)
+[![javadoc](https://javadoc.io/badge2/org.kiwiproject/elucidation/javadoc.svg)](https://javadoc.io/doc/org.kiwiproject/elucidation)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![Maven Central](https://img.shields.io/maven-central/v/org.kiwiproject/elucidation)](https://central.sonatype.com/artifact/org.kiwiproject/elucidation/)
```

Comment:
Do we want to put this here yet? Given that there is no published version yet from us?

Reply:
It should just say nothing found or something like that.  That way I won't forget.